Xk6 disruptor update install instructions (#916)
grafana/k6-docs: src/data/markdown/docs/40 xk6-disruptor/01 Get started/03 Installation.md

xk6-disruptor is a [k6 extension](/extensions). You have to run a k6 version built with the disruptor extension to use the [disruptor APIs](/javascript-api/xk6-disruptor/api/) in your k6 tests.

The following sections explain the different options to get this custom binary.

## Download a release binary

The quickest way to get started is to [download a release binary from GitHub](https://github.com/grafana/xk6-disruptor/releases).

## Build from source

> This option allows you to build a k6 binary using [xk6](https://github.com/grafana/xk6).

Reviewer: ```suggestion
You can also build a k6 binary using [xk6](https://github.com/grafana/xk6).
```
